transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -75,6 +78,14 @@ public void run() {
         logger.trace("New EventLoop: {}", this.toString());
     }
 
+    private static Queue<Runnable> newTaskQueue(
+            EventLoopTaskQueueFactory queueFactory) {
+        if (queueFactory == null) {
+            return newTaskQueue0(DEFAULT_MAX_PENDING_TASKS);
+        }
+        return queueFactory.newTaskQueue(DEFAULT_MAX_PENDING_TASKS);
+    }
```

Comment:
move to `protected static` in superclass? (since used by all the impls it looks like?)

Reply:
let us keep it here for now as the usage of a MPSC queue that is non-blocking is specific to some event loop impls